themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml

```diff
@@ -33,30 +33,48 @@
 <?php if ($hasSubLinks): ?>
   <div class="<?=implode(' ', $classList) ?>">
 <?php else: ?>
-  <a href="<?=$toggleUrl ?>" class="<?=implode(' ', $classList) ?>" data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"<?php if($this->facet['isApplied']): ?> title="<?=$this->transEscAttr('applied_filter') ?>"<?php endif;?> data-lightbox-ignore>
+  <a href="<?=$toggleUrl ?>"
+     class="<?=implode(' ', $classList) ?>"
+     data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>"
+     data-count="<?=$this->facet['count'] ?>"
+     title="<?php if($this->facet['isApplied']): ?><?=$this->transEsc('applied_filter') ?> - <?=$this->transEsc('page_reload_on_deselect_hint', ['%%filter_name%%' => $this->facet['displayText']])?><?php else: ?><?=$this->transEsc('page_reload_on_select_hint', ['%%filter_name%%' => $this->facet['displayText']])?>
```

Comment:
This attribute might be more readable if we just assign the title in a PHP block above the link -- as it is here, the wrapping is a bit confusing.